src/Pact/Native.hs

```diff
@@ -56,14 +56,16 @@ module Pact.Native
     , cdPrevBlockHash
     ) where
 
-import Control.Arrow hiding (app)
+import Control.Arrow hiding (app, first)
 import Control.Exception.Safe
 import Control.Lens hiding (parts,Fold,contains)
 import Control.Monad
 import Control.Monad.IO.Class
 import qualified Data.Attoparsec.Text as AP
+import Data.Bifunctor (first)
```

Comment:
This doesn't conflict with the import of `first` from `Control.Arrow` just above?

Reply:
`import Control.Arrow hiding (app, first)`
Added it to the `hiding` so I could have my bifunctor `first`.